Docker/test/selenium/package.json

```diff
@@ -20,7 +20,6 @@
 		"stylelint-config-wikimedia": "0.10.1",
 		"wdio-mediawiki": "^1.0.0",
 		"wdio-wikibase": "^5.1.0",
-		"webdriverio": "^6.1.16",
```

Comment:
https://github.com/wmde/wikibase-release-pipeline/pull/211/checks?check_run_id=3487116732#step:4:1307

still there

Reply:
I found this in `package-lock.json`. I think it's the reason why `webdriverio-6.8.0` appeared in the test

![Screenshot from 2021-09-02 12-36-26](https://user-images.githubusercontent.com/17342474/131829263-1a8bc2bb-e675-4fb6-938c-f4889aa1559b.png)

Reply:
line 6902

Reply:
I think what was required is this -> https://github.com/wmde/wikibase-release-pipeline/pull/212